operatorconfig/moduleconfig/authorization/v2.0.0-alpha/deployment.yaml

```diff
@@ -416,6 +416,116 @@ spec:
     kind: Issuer
     group: cert-manager.io
 ---
+# Controller
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: controller
```

Comment:
Can we name this as "authorization-controller" instead of just controller?

Reply:
Renamed it to authorization-controller in all the places.